components/script/dom/webidls/DissimilarOriginWindow.webidl

```diff
@@ -13,7 +13,7 @@
 // way to enforce security policy.
 
 // https://html.spec.whatwg.org/multipage/#window
-[Global, Exposed=(Window,DissimilarOriginWindow), LegacyNoInterfaceObject]
+[Global=(Window,DissimilarOriginWindow), Exposed=(Window,DissimilarOriginWindow), LegacyNoInterfaceObject]
```

Comment:
Why is this change necessary?

Reply:
More specifically, why not `Global=DissimilarOriginWindow`?

Reply:
Ah thanks. The new webidl.py raised this error so I added the identifier list:
```
    File "/Users/wuwayne/Desktop/servo/third_party/WebIDL/WebIDL.py", line 2059, in addExtendedAttributes
      raise WebIDLError(
  WebIDL.WebIDLError: error: [Global] must either take an identifier or take an identifier list, /Users/wuwayne/Desktop/servo/components/script/dom/bindings/codegen/../../webidls/DissimilarOriginWindow.webidl line 16:1
  [Global, Exposed=(Window,DissimilarOriginWindow), LegacyNoInterfaceObject]
   ^
 ```

Your suggestion is better and don't require fixing CodegenRust.py. I'll update it.